libs/ui/elements/src/lib/metadata-info/metadata-info.component.html

```diff
@@ -1,9 +1,19 @@
-<p
-  class="text-[28px] text-title text-center mb-6 font-title sm:text-left"
-  translate
->
-  record.metadata.about
-</p>
+<div class="flex flex-row justify-between">
+  <p
+    class="text-[28px] text-title text-center mb-6 font-title sm:text-left"
+    translate
+  >
+    record.metadata.about
+  </p>
+  <div *ngIf="metadata.status" class="flex items-start pt-3 pr-4">
+    <span
+      class="inline-flex items-center justify-center px-2 py-1 text-13 uppercase font-medium leading-none bg-primary text-white rounded opacity-40"
+      translate
+    >
+      domain.record.status.{{ metadata.status }}
```

Comment:
Should this be part of this PR? I'm wondering where this is being displayed.

Reply:
Oh, that must be something from my other PR that slipped in. Will remove it, thanks!